cypress/Objects/LoginpageOpjects.js

```diff
@@ -0,0 +1,59 @@
+class LoginPage{
+    visit() {
+        cy.visit('/')
+    }
+
+    getUserValidationIndicator(){
+        return cy.get('.Select.not-valid')
+    }
+
+    getLoginButton(){
+        return cy.get('[type="submit"]')
+    }
+
+    getLoginFormUserId(){
+        return cy.get('[id="loginForm.userId"]')
+    }
+
+    getUserName(userName){
+        return cy.get('[aria-label="'+ userName +'"]')
+    }
+
+    getLoginFormRoleId(){
+        return cy.get('[id="loginForm.role"]')
+    }
+
+    getUserRole(){
+        return  cy.get('[aria-label="Team Lead"]')
+    }
+
+    getSubmitButton(){
+        return cy.get('[type="submit"]')
+    }
+
+    getUrl(){
+        return cy.url().should('include', '/time-logging')
+    }
+
+    getPageTitle(){
+       return cy.get('.page__title').contains('Timesheets')  
+    }
+
+    getCalendarIsVisible(){
+        return cy.get('.calendar')
```

Comment:
You are not checking if the calendar is visible in this place, thus the name of the function is incorrect